python/src/v3/board.py

```diff
@@ -22,6 +22,7 @@ def print_summary(klass, board_data):
         print('--------------------')
 
     # provides a human-readable identifier for a board
+    @classmethod
     def text_id(klass, board_data):
```

Comment:
By general convention, the first argument to a class method is spelled `cls`.

Reply:
Thanks. I'll change all instances of klass to cls.